plutus-playground/plutus-playground-client/src/Action.purs

```diff
@@ -69,7 +69,7 @@ actionPane index action =
                 [ text $ show (index + 1) ]
             , button
                 [ classes [ btn, btnInfo, pullRight ]
-                , onClick $ input_ $ RemoveAction index
+                , onClick $ input_ $ ModifyActions $ RemoveAction index
```

Comment:
in Haskell I would prefer `onClick . input_ . ModifyActions . RemoveAction $ index` but I'm not sure about `onClick <<< input_ <<< ModifyActions <<< RemoveAction $ index`

Reply:
I this this just a question of familiarity. To me they read the same (even though yes, I'd prefer it if PS used the briefer syntax).